portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java

```diff
@@ -122,7 +122,9 @@ public AssetTag addTag(
 		List<AssetTag> tags = new ArrayList<>();
 
 		for (String name : names) {
-			AssetTag tag = fetchTag(group.getGroupId(), name);
+
+			String nameValid = StringUtil.toLowerCase(name.replaceAll("\\s+",""));
```

Comment:
Use StringUtil to trim string

Reply:
I updated it. Please check it again.
Thanks.